README.rst

```diff
@@ -183,6 +183,19 @@ index in your Windows, Linux, or Mac file system::
 Note the triple ``///`` after ``file:`` -- two for the protocol,
 the third for the root of the local file system.
 
+How to get multiple links/indexs working?
+-----------------------------------------
+
+First, add this to your `~/.config/pip/pip.config` file:
+
+    [global]
+    timeout = 3
```

Comment:
Because otherwise you get a X second delay for any module that you do not have in your pip2pi local store?

Reply:
An `X` seconds delay, and at least five retries (default).
But more importantly, this tip belongs to the `pip` documentation (which is already there) rather than `pip2pi` documentation. Don't you think?

Reply:
Yes, that is fair.